packages/db-objects-file/lib/objects/objectsInMemServerRedis.js

```diff
@@ -757,24 +723,29 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
                             notExists: true
                         }];
                     }
-                    const response = [];
-                    const baseName = name + ((!name.length || name.endsWith('/')) ? '' : '/');
-                    res.forEach(arr => {
-                        let entryId = id;
-                        if (arr.isDir) {
-                            if (entryId === '' || entryId === '*') {
-                                entryId = arr.file;
-                                arr.file = '_data.json'; // We return a "virtual file" to mark the directory as existing
-                            } else {
-                                arr.file += '/_data.json'; // We return a "virtual file" to mark the directory as existing
-                            }
+                } catch (err) {
+                    if (!err.toString().endsWith(utils.ERRORS.ERROR_NOT_FOUND)) {
+                        return void handler.sendError(responseId, new Error(`ERROR readDir id=${id}: ${err.message}`));
```

Comment:
That seems to confirm my argument that you rather want to check the error message instead of toString